packages/contracts-bedrock/test/kontrol/pausability-lemmas.md

```diff
@@ -202,17 +202,17 @@ The summary lemma is as follows, with commentary inlined:
        // Various well-formedness constraints. In particular, the maxBytesLength-related ones are present to
        // remove various chops that would otherwise creep into the execution, and are reasonable since byte
        // arrays in actual programs would never reach that size.
-       andThenBool 0 <=Int PCOUNT
-       andThenBool 0 <=Int LENGTH andThenBool LENGTH <Int maxBytesLength
-       andThenBool 0 <=Int SRC    andThenBool SRC    <Int maxBytesLength
-       andThenBool 0 <=Int DEST   andThenBool DEST   <Int maxBytesLength
-       andThenBool #sizeWordStack(WS) <=Int 1015
-       andThenBool SRC +Int LENGTH <=Int DEST // No overlap between source and destination
-       andThenBool DEST <=Int lengthBytes(LM) // Destination starts within current memory
-       andThenBool PCOUNT +Int 51 <Int lengthBytes(JUMPDESTS) // We're not look outside of the JUMPDESTs bytearray
+       andBool 0 <=Int PCOUNT
```

Comment:
What’s the difference between `andThenBool` vs `andBool`?

Reply:
This was introduced to silence an error that appeared when Kontrol tried to access an array outside of its bounds. This error was caused by the `requires` clauses not being applied in order since the bound constraint was introduced before the array access. `andThenBool` was put in place to force the ordering of the constraints. However, this should have been ordered automatically. The error appears to be showing no longer.
Also, note that the error of out-of-bounds array access did not affect the soundness of the proof. It just made the summary not apply, which shouldn't have applied anyway in those cases.